locale/pt-br/blog/announcements/nodejs-certified-developer-program.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Os Parceiros da Fundação Node.js com a Fundação Linux no Novo Programa de Certificação Node.js
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
```

Comment:
Category is translated ?

Reply:
I think they are so they're shown in the repo, but I'm not sure. @Trott do you know something about it?

Reply:
Nope, shouldn't be translated.